lineman/app/modules/navbar/navbar_search_items/navbar_search_comment.haml

```diff
@@ -1,2 +1 @@
-.navbar-search-comment
-  {{comment.body}}
\ No newline at end of file
+.navbar-search-comment{btf-markdown: 'result.blurb'}
```

Comment:
Love to see all this experimentation. Thank you so much. This looks really
cool.

On Sat, Feb 7, 2015 at 10:32 AM, James Kiesel <notifications@github.com>
wrote:

> In
> lineman/app/modules/navbar/navbar_search_items/navbar_search_comment.haml
> <https://github.com/loomio/loomio/pull/1873#discussion_r24273679>:
>
> > \ No newline at end of file
> > +.navbar-search-comment{btf-markdown: 'result.blurb'}
>
> Haven't done this all the way out the frontend yet, but can verify that we
> can handle rendering blurbs by making em markdown, woo!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/1873/files#r24273679>.
>